src/content/en/updates/posts/2016/02/devtools-digest-supercharged-remote-debugging.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: updates/post
+title: "Supercharged Remote Rebugging, Class Toggles and our own late night show?!"
+published_on: 2016-02-25
+updated_on: 2016-02-25
+authors:
+  - pbakaus
+tags:
```

Comment:
What do these do? Are they only for web/updates, or do they apply to web/tools also?